packages/metadata-common/src/MetadataEditor.tsx

```diff
@@ -100,6 +101,22 @@ export class MetadataEditor extends ReactWidget {
         if (!this.name) {
           this.title.label = `New ${this.schemaDisplayName}`;
         }
+        // Find categories of all schema properties
+        this.categories = { _noCategory: [] };
```

Comment:
Since this is now a dict containing all the fields this name isn't the most accurate anymore. Perhaps `schemaPropertyByCategory`, `sortedSchemaProperties`, or `categorizedSchemaProperties`. All are a bit wordy but better show that it is a dict of schema properties organized by category